### ISSUE TITLE
Python command not found

### ISSUE THREAD

**Issue by zevlag** [jprochazka/adsb-exchange#22]
I began by installing this image:
http://piaware.flightcdn.com/piaware-sd-card-3.5.0.img.zip

Then ran the script from this repo:

```sudo apt-get -y update
sudo apt-get -y install git
git clone https://github.com/jprochazka/adsb-exchange.git
cd adsb-exchange
chmod +x setup.sh
sudo ./setup.sh
cd ..
```

I get an error after doing that:

```pi@piaware:~/adsb-exchange$ sudo ./setup.sh

Checking for packages needed to run this script...
Installing the curl package...

Hit http://flightaware.com jessie InRelease
Hit http://mirrordirector.raspbian.org jessie InRelease                                                               
Hit http://archive.raspberrypi.org jessie InRelease                                                                   
Hit http://flightaware.com jessie/piaware armhf Packages                                                              
Hit http://mirrordirector.raspbian.org jessie/main armhf Packages        
Hit http://mirrordirector.raspbian.org jessie/contrib armhf Packages     
Hit http://mirrordirector.raspbian.org jessie/non-free armhf Packages
Hit http://mirrordirector.raspbian.org jessie/firmware armhf Packages
Hit http://archive.raspberrypi.org jessie/main armhf Packages     
Hit http://archive.raspberrypi.org jessie/ui armhf Packages
Ign http://flightaware.com jessie/piaware Translation-en_US                                                            
Ign http://flightaware.com jessie/piaware Translation-en       
Ign http://mirrordirector.raspbian.org jessie/contrib Translation-en_US                                                                             
Ign http://mirrordirector.raspbian.org jessie/contrib Translation-en                                                                                
Ign http://mirrordirector.raspbian.org jessie/firmware Translation-en_US                                                                            
Ign http://mirrordirector.raspbian.org jessie/firmware Translation-en                                                                               
Ign http://mirrordirector.raspbian.org jessie/main Translation-en_US                                                                                
Ign http://mirrordirector.raspbian.org jessie/main Translation-en                                                                                   
Ign http://mirrordirector.raspbian.org jessie/non-free Translation-en_US                                                                            
Ign http://archive.raspberrypi.org jessie/main Translation-en_US                                                                                    
Ign http://mirrordirector.raspbian.org jessie/non-free Translation-en                                                                               
Ign http://archive.raspberrypi.org jessie/main Translation-en                                                                                       
Ign http://archive.raspberrypi.org jessie/ui Translation-en_US                                                                                      
Ign http://archive.raspberrypi.org jessie/ui Translation-en                                                                                         
100% [Packages 50.1 MB]                                                                                                                             
100% [Packages 50.1 MB]
Reading package lists... Done
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following extra packages will be installed:
  libcurl3
The following NEW packages will be installed:
  curl libcurl3
0 upgraded, 2 newly installed, 0 to remove and 39 not upgraded.
Need to get 427 kB of archives.
After this operation, 753 kB of additional disk space will be used.
Get:1 http://mirrordirector.raspbian.org/raspbian/ jessie/main libcurl3 armhf 7.38.0-4+deb8u5 [232 kB]
Get:2 http://mirrordirector.raspbian.org/raspbian/ jessie/main curl armhf 7.38.0-4+deb8u5 [196 kB]
Fetched 427 kB in 1s (225 kB/s)   
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package libcurl3:armhf.
(Reading database ... 21648 files and directories currently installed.)
Preparing to unpack .../libcurl3_7.38.0-4+deb8u5_armhf.deb ...
Unpacking libcurl3:armhf (7.38.0-4+deb8u5) ...
Selecting previously unselected package curl.
Preparing to unpack .../curl_7.38.0-4+deb8u5_armhf.deb ...
Unpacking curl (7.38.0-4+deb8u5) ...
Processing triggers for man-db (2.7.0.2-5) ...
Setting up libcurl3:armhf (7.38.0-4+deb8u5) ...
Setting up curl (7.38.0-4+deb8u5) ...
Processing triggers for libc-bin (2.19-18+deb8u7) ...

./setup.sh: line 72: python: command not found
(23) Failed writing body
pi@piaware:~/adsb-exchange$ sudo reboot
```

**Comment by zevlag** [comment]
Just changing this to be python3 doesn't fix the problem....

```  File "<string>", line 1
    import json,sys;obj=json.load(sys.stdin);print obj['results'][0]['elevation'];
                                                     ^
SyntaxError: invalid syntax```

**Comment by zevlag** [comment]
This is fixed by installing python 2.7   See: https://github.com/jprochazka/adsb-exchange/pull/24
